source/includes/connection-snippets/x509-snippets/java-connection.java

```diff
@@ -0,0 +1,18 @@
+// begin x509 connection
+import com.mongodb.client.MongoClient;
+import com.mongodb.client.MongoClients;
+
+public class X509Connection {
+
+public static void main(String args[]) {
+        // Set system properties via commandline or programmatically
+        System.setProperty("javax.net.ssl.keyStore", "/etc/certs/mongodb/v3/client.keystore");
+        System.setProperty("javax.net.ssl.keyStorePassword", "<your_password>");
```

Comment:
@jyemin should this use tls URI options instead (per [DRIVERS-557](https://jira.mongodb.org/browse/DRIVERS-557))?